src/openforms/validations/registry.py

```diff
@@ -84,6 +87,7 @@ def decorator(validator: Union[Type, ValidatorType]):
                 identifier=identifier,
                 verbose_name=verbose_name,
                 callable=call,
+                components=getattr(validator, "components", ()),
```

Comment:
I would enforce that validators must specify this at all times and get rid of the `getattr` stuff.